src/evm_transition_tool/transition_tool.py

```diff
@@ -407,3 +437,68 @@ def calc_state_root(
         if state_root is None or not isinstance(state_root, str):
             raise Exception("Unable to calculate state root")
         return new_alloc, bytes.fromhex(state_root[2:])
+
+    def verify_fixture(
```

Comment:
Let's move this to the geth interface and add an abstract method here that raises an Exception if the instantiated class doesn't implement it.